### DISCUSSION TITLE
xtb --ohess optimization with gbsa doesn't converge for NMA/THF system

### DISCUSSION THREAD

**Discussion by lxsep** [grimme-lab/xtb#505] [Q&A]
I have a system of one molecule of NMA in a cluster of 17 molecules of THF and am also trying to use the --gbsa THF implicit solvent and calculate the frequencies with --ohess. The problem is that the optimizations does not converge, the energy goes down and then up suddenly and the calculation crashes, I have changed the etemp to 500k and even to 1000k and still does not converge. the list of errors goes like this:

[ERROR] Program stopped due to fatal error
-7- Geometry optimization failed
-6- xtb_geoopt: Trying to recover from failed geometry optimization
-5- xtb_geoopt: Geometry optimization did not converge
-4- optimizer_ancopt: Could not relax structure
-3- optimizer_relax: SCF not converged, aborting...
-2- xtb_calculator_singlepoint: Electronic structure method terminated
-1- scf: Self consistent charge iterator did not converge

Is there a way I could make this calculation work?

**Comment by awvwgk** [comment]
Can't really give advice on this without knowing the actual input. One thing you might try is using `--alpb THF` instead or changing the Hamiltonian `--gfn 1`.
